client.py

```diff
@@ -0,0 +1,20 @@
+import socket
+import sys
+
+from kvstore.constants import INPUT_PROMPT
+
+def main():
+    while True:
+        i = input(INPUT_PROMPT)
+
+        s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        s.connect(('localhost', 65432))
+        s.send(i.encode(encoding='UTF-8',errors='strict'))
+
+        response = s.recv(1024)
```

Comment:
how do i receive a response greater than 1024 bytes?

Reply:
Just need to do this in a loop :)